src/Tests/Microsoft.NET.Build.Tests/GivenThatWeWantToCopyLocalDependencies.cs

```diff
@@ -0,0 +1,153 @@
+
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.NET.Build.Tests
+{
+    public class GivenThatWeWantToCopyLocalDependencies : SdkTest
+    {
+        public GivenThatWeWantToCopyLocalDependencies(ITestOutputHelper log) : base(log)
+        {
+        }
+
+        [Fact]
+        public void It_copies_local_package_dependencies_on_build()
+        {
+            const string ProjectName = "TestProjWithPackageDependencies";
+
+            TestProject testProject = new TestProject()
+            {
+                Name = ProjectName,
+                IsSdkProject = true,
+                TargetFrameworks = "netcoreapp3.0",
+                IsExe = true
+            };
+
+            testProject.PackageReferences.Add(new TestPackageReference("Newtonsoft.Json", "11.0.2"));
+            testProject.PackageReferences.Add(new TestPackageReference("sqlite", "3.13.0"));
+
+             var testProjectInstance = _testAssetsManager
+                .CreateTestProject(testProject)
+                .Restore(Log, ProjectName);
+
+            var buildCommand = new BuildCommand(Log, testProjectInstance.TestRoot, ProjectName);
+
+            buildCommand.Execute()
+                .Should()
+                .Pass();
+
+            var outputDirectory = buildCommand.GetOutputDirectory(testProject.TargetFrameworks);
+            outputDirectory.Should().OnlyHaveFiles(new[] {
+                $"{ProjectName}{Constants.ExeSuffix}",
+                $"{ProjectName}.deps.json",
+                $"{ProjectName}.dll",
+                $"{ProjectName}.pdb",
+                $"{ProjectName}.runtimeconfig.dev.json",
+                $"{ProjectName}.runtimeconfig.json",
+                "Newtonsoft.Json.dll",
+                "runtimes/linux-x64/native/libsqlite3.so",
+                "runtimes/osx-x64/native/libsqlite3.dylib",
+                "runtimes/win7-x64/native/sqlite3.dll",
+                "runtimes/win7-x86/native/sqlite3.dll"
+            });
+        }
+
+        [Fact]
+        public void It_does_not_copy_local_package_dependencies_when_requested_to()
+        {
+            const string ProjectName = "TestProjWithPackageDependencies";
+
+            TestProject testProject = new TestProject()
+            {
+                Name = ProjectName,
+                IsSdkProject = true,
+                TargetFrameworks = "netcoreapp3.0",
+                IsExe = true
+            };
+
+            testProject.PackageReferences.Add(new TestPackageReference("Newtonsoft.Json", "11.0.2"));
+            testProject.PackageReferences.Add(new TestPackageReference("sqlite", "3.13.0"));
+
+             var testProjectInstance = _testAssetsManager
+                .CreateTestProject(testProject)
+                .Restore(Log, ProjectName);
+
+            var buildCommand = new BuildCommand(Log, testProjectInstance.TestRoot, ProjectName);
+
+            buildCommand.Execute("-p:CopyLocalLockFileAssemblies=false")
```

Comment:
Instead of passing this in as a command-line argument, I'd suggest:

```C#
testProject.AdditionalProperties["CopyLocalLockFileAssemblies"] = "false"
```

(I think I got the member names etc right on that).

Reply:
I'll fix.